lua/org-bullets.lua

```diff
@@ -128,8 +131,8 @@ local function get_ts_positions(bufnr, start_row, end_row, root)
     [[
       (stars) @stars
       (bullet) @bullet
-      ((expr) @_item (#eq? @_item "[-]")) @cancelled
       ((expr) @_done (#eq? @_done "[X]")) @done
+      ((expr) @_half (#eq? @_half "[-]")) @half
```

Comment:
I would avoid changing this here as I'm very actively working on these queries, and locally they are all different anyway, so if you could revert this part of the change that'd be great 🙏🏿